### DISCUSSION TITLE
Why is the SvelteKit output directory set to "public" by default?

### DISCUSSION THREAD

**Discussion by danawoodman** [vercel/community#1641] [Help] [answered]
### Summary

As far as I can tell, the `SvelteKit` deployment defaults have `public` set, despite the fact the default for a new SvelteKit project using adapter-auto is `.svelte-kit`. 

Perhaps the defaults can be updated to work out of the box with SvelteKit?

### Additional information

1. Create new SK app
2. Create Github repo
3. Add in Vercel
4. Use default settings for SvelteKit
5. See your build break until you set `Output Directory` to `.svelte-kit`

**Comment by amyegan** [comment, marked as answer]
I'm not able to reproduce the problem, so I may be misunderstanding one of your repro steps. Are you able to share your example app so I can get a better look at what is happening?

**Reply by danawoodman** [reply]
Sorry, guess this was fixed in Kit somewhat recently and I was running an older version, ignore me

Possibly related: https://github.com/sveltejs/kit/issues/3142
